pricejar/templates/navbar.html

```diff
@@ -1,41 +1,177 @@
+              id="dropdownId"
+              data-bs-toggle="dropdown"
+              aria-haspopup="true"
+              aria-expanded="false"
+              >Store</a
+            >
+            <div class="dropdown-menu" aria-labelledby="dropdownId">
+              <a class="dropdown-item" href="#">Action 1</a>
+              <a class="dropdown-item" href="#">Action 2</a>
+            </div>
+          </li>
+          <li class="nav-item dropdown">
+            <a
+              class="nav-link dropdown-toggle"
+              href="#"
+              id="dropdownId"
+              data-bs-toggle="dropdown"
+              aria-haspopup="true"
+              aria-expanded="false"
+              >Categories</a
+            >
+            <div class="dropdown-menu" aria-labelledby="dropdownId">
+              <a class="dropdown-item" href="#">Action 1</a>
+              <a class="dropdown-item" href="#">Action 2</a>
+            </div>
+          </li>
+        </ul>
+        <button class="btn btn-outline-success my-2 my-sm-0" type="submit">
+          Check Now
+        </button>
+      </div>
+    </div>
+  </nav>
+  {%endif%}
+</header>
+
+<!-- Nav -->
+<!-- navbar for authenticated user  -->
+
+{% if request.user.is_authenticated %}
+<nav <!-- Nav -->
+  <nav
+    class="navbar fixed-top navbar-expand-lg navbar-light"
+    style="background-color: #ffff"
+  >
+    <div class="container-fluid">
+      <a class="navbar-brand" href="#" style="padding: 10px">
+        <span style="color: #801b4b">P</span>
+        <span style="color: #801b4b">R</span>
+        <span style="color: #801b4b">I</span>
+        <span style="color: #f2b109">C</span>
+        <span style="color: #f2b109">E</span>
+        <span style="color: black">J</span>
+        <span style="color: black">A</span>
+        <span style="color: black">R</span>
+      </a>
+      <button
+        class="navbar-toggler ms-auto"
+        type="button"
+        data-bs-toggle="collapse"
+        data-bs-target="#navbarSupportedContent"
+        aria-controls="navbarSupportedContent"
+        aria-expanded="false"
+        aria-label="Toggle navigation"
+      >
+        <span class="navbar-toggler-icon"></span>
+      </button>
+      <div
+        class="collapse navbar-collapse justify-content-center"
+        id="navbarSupportedContent"
+      >
+        <ul class="navbar-nav me-auto mb-2 mb-lg-0 align-items-end">
+          <li class="nav-item">
+            <a class="nav-link active" aria-current="page" href="{%url 'home'%}"
+              >Home</a
+            >
+          </li>
+
+          <li class="nav-item dropdown">
+            <a
+              class="nav-link dropdown-toggle"
+              href="#"
+              id="navbarDropdown"
+              role="button"
+              data-bs-toggle="dropdown"
+              aria-expanded="false"
+            >
+              Categories
+            </a>
+            <ul class="dropdown-menu" aria-labelledby="navbarDropdown">
```

Comment:
I think the drop down here are now updated. Check the latest copy of home or any page containing navbar. Compare and use the latest.